public/api/v2/notes/banner

```diff
@@ -0,0 +1,14 @@
+{
+    "id": "ABC123",
+    "alias": "banner",
+    "lastChange": {
+        "userId": "snskxnksnxksnxksnx",
+        "username": "testy",
+        "timestamp": 123456789
+    },
+    "viewcount": 0,
+    "createtime": "2020-05-22T20:46:08.962Z",
+    "content": "This is the test banner text",
+    "authorship": [],
+    "preVersionTwoNote": true
```

Comment:
This should be false.

Reply:
As this is just a mock call not really necessary to change, but technically correct

Reply:
If this stays false, I can definitely live with it, but since we should touch the code again anyway, why not fix it all along?

Reply:
I don't get what the problem is.

Reply:
@mrdrogdrog As this technically is not a preVersionTwoNote the field should be false, but I just copied the `old` note which is (like the name indicates) supposed to be an old pre 2.0 note.
I'll change that quickly.

Reply:
Oh okay. I get it. The problem isn't the attribute itself, but the value of the attribute in the banner note.